frontend/src/components/grantaccess/GrantAccess.vue

```diff
@@ -494,7 +500,7 @@ function removeForestClientFromList(index: number) {
                                     :validateOnChange="true"
                                     v-bind="field"
                                     class="w-100 custom-input"
-                                    @input="resetForestClientNumberData()"
+                                    @input="cleanupForestClientNumberInput()"
```

Comment:
we only need to cleanup the forest client number input field when input changes, but will keep the forest client card